modules/assim.sequential/R/Analysis_sda_multiSite.R

```diff
@@ -361,9 +419,9 @@ GEF.MultiSite<-function(setting, Forecast, Observed, H, extraArg,...){
       valueInCompiledNimbleFunction(Cmcmc$samplerFunctions[[samplerNumberOffset+i]], 'toggle', 1-y.ind[i])
     }
     
-    
+    save(inits.pred, dimensions.tobit, constants.tobit, data.tobit,  model_pred, conf, Rmcmc, Cmodel, Cmcmc, file="SDA/NimbleVars.RData" )
```

Comment:
1) is the SDA folder guaranteed to exist here?
2) Is this guaranteed to be writing to the correct workflow folder?
In general, I prefer that we write to full paths, not relative ones.